docs/process/metrics.md

```diff
@@ -0,0 +1,52 @@
+# GitHub Desktop Metrics
+Desktop is a free, open source product that is maintained and worked on by the small, distributed team of @desktop/core, @desktop/comrades, and the general open-source community. As a result, we do not have the resources to do continuous, detailed user studies of GitHub Desktop users and usage. In other words, we love you, we want to build the right features to help you, and yet we don't know enough about you as a whole to make informed prioritization decisions.
+
+We measure certain events in the GitHub Desktop app in order to learn more about how you are using the app. This helps us prioritize features and bugs and understand what things are most important to the largest number of people. Our understanding of these metrics is supplemented by some user research, usability testing, and feedback from the community. This document is intended to provide information about each thing we're measuring and what we're using it for.
+
+You can always see our recently released user-facing features and fixes in the [release notes](https://desktop.github.com/release-notes/) of our marketing page.
+
+## Dimensions
+These are general metrics about users that are aggregated to understand general user behavior and to help us segment usage patterns. These helps us prioritize work, predict technical and performance implications of our features, and plan for devOps needs.
+
+| Metric | Description | Justification |
+|:--|:--|:--|
+| `version` | The version of Desktop. | To visualize update rates and performance metrics for each version of Desktop, so we can track whether users are staying up-to-date and which older versions are active. |
+| `osVersion` | The OS version. | To identify the most common versions of operating systems people use Desktop on, so we can more accurately prioritize version-specific bugs. |
+| `platform` | The OS. | To understand which platforms are most popular among people who use Desktop, so we can more accurately prioritize platform-specific bugs.  |
+| `repositoryCount` | The total number **tracked** of repositories **in Desktop**. | To understand the typical number of repositories tracked in Desktop, so we can make appropriate decisions in terms of performance and UI. |
```

Comment:
Is there a reason why this is the only use of bold? It feels out of place. Also, do we mean "The total number of tracked repositories in Desktop."

Reply:
I think it was to emphasize repositories that Desktop is tracking. As opposed to the amount of repos you have on disk or on GitHub. But, I'll remove the emphasis.